Console/retro/Console.cc

```diff
@@ -154,10 +234,11 @@ void Console::Draw(Rect r)
 
     short minRow = std::max(0, (r.top - bounds.top) / cellSizeY);
     short maxRow = std::min((int)rows, (r.bottom - bounds.top + cellSizeY - 1) / cellSizeY);
-    
-    short minCol = std::max(0, (r.left - bounds.left) / cellSizeX);
-    short maxCol = std::min((int)cols, (r.right - bounds.left + cellSizeX - 1) / cellSizeX);
-    
+
+    short minCol = 0;// std::max(0, (r.left - bounds.left) / cellSizeX);
```

Comment:
Probably a leftover from the width calculation code?

Reply:
Yes, it is, I think we can use:
~~~~
short maxCol = std::min((int)cols, (r.right - bounds.left + cellSizeX - 1) / cellSizeX);
~~~~

Reply:
Yes, indeed, it's a leftover, the original code should be more efficient.